jwala-services/src/main/java/com/cerner/jwala/service/group/impl/GroupJvmControlServiceImpl.java

```diff
@@ -53,19 +54,25 @@ public void controlAllJvms(final ControlGroupJvmRequest controlGroupJvmRequest,
                 jvms.addAll(groupsJvms);
             }
         }
-
+        LOGGER.info("jvm size: " + jvms.size());
```

Comment:
Need more detail what you are logging here (or remove...). If I see this in a log: what does it mean?